build/crds/edgecluster/edgecluster_v1.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       type: date
       JSONPath: .status.lastheartbeat
     - name: Healthy
-      type: boolean
+      type: string
```

Comment:
string is definitely more descriptive than boolean. with this, the name and description should be changed to something like HealthStatus to indicate it's not a true/false

Reply:
Agree. Updated.